lib/mayaUsd/ufe/UsdUndoDuplicateSelectionCommand.cpp

```diff
@@ -155,6 +181,16 @@ void UsdUndoDuplicateSelectionCommand::execute()
                         }
                     }
                 }
+            };
+
+            // Cleanup relationships and connections on the duplicate.
+            if (_dstParentItem) {
+                auto p = stage->GetPrimAtPath(duplicatePair.second);
+                updatePropertiesAndRelationship(p);
+            } else {
+                for (auto p : UsdPrimRange(stage->GetPrimAtPath(duplicatePair.second))) {
```

Comment:
I did some tests here since when I tried to update the paths using the `updatePropertiesAndRelationship `function, it updates also the paths obtained by the subtree in `UsdPrimRange`.

In the clipboard case, we are copying to a different destination layer and I got incorrect paths for the subtree (tried with nested compounds). 

I tried avoiding updating them using `updatePropertiesAndRelationship` and I got correct results, so I've investigated and I think the `SdfCopySpec` function we call in the `UsdUndoDuplicateCommand::execute` takes care of the subtree updates. From the [doc](https://openusd.org/dev/api/copy_utils_8h.html#ad20defdb6654ee0937942116177fd4cc):
" Attribute connections, relationship targets, inherit and specializes paths, and internal sub-root references that target an object beneath srcPath will be remapped to target objects beneath dstPath. "
 
I did other tests for the `updatePropertiesAndRelationship` function for the `UsdUndoDuplicateSelectionCommand `
( calling it in "standard" not clipboard case), and I saw the subtrees had always updated paths, so I'm not sure if we need to call it on the `UsdPrimRange`.